### ISSUE TITLE
Unselected removed items are deleted using admin delete function

### ISSUE THREAD

**Issue by qubez** [nilsteampassnet/TeamPass#1507]
### Steps to reproduce
1. Delete several items and several folders,
2. Log in as admin, go to utilities (views) -> deletions
3. select some folders, leaving some items unchecked
4. press delete
### Expected behaviour

Only selected checked items should be removed
### Actual behaviour

All items are removed even if none are checked

![teampass-unselectedaredeleted1](https://cloud.githubusercontent.com/assets/1425520/18652620/b73ab708-7e88-11e6-8021-85a948b89f86.png)
(press delete with the items checked as shown, then we get:)
![teampass-unselectedaredeleted2](https://cloud.githubusercontent.com/assets/1425520/18652632/ca228dfa-7e88-11e6-98f1-266dc412bd34.png)
### Server configuration

**Operating system**:
Ubuntu 16.04.2 32 bit

**Web server:**
Apache/2.4.18 (Ubuntu) OpenSSL/1.0.2g-fips

**Database:**
MySQL 5.7.13-0ubuntu0.16.04.2 - (Ubuntu) 

**PHP version:**
PHP Version 7.0.8-0ubuntu0.16.04.2

**Teampass version:**
Teampass development branch

**Updated from an older Teampass or fresh install:**
Updated from 2.1.26 (17) w script
### Client configuration

**Browser:**
Firefox 48 32 Bit
**Operating system:**
Windows 7 x64

**Comment by nilsteampassnet** [comment]
Could it be that the items where in the selected folders ? 
In this case it is normal as when deleting a folder, related items are deleted. 

Now Ive checked on my own and reviewed the code. I couldn't identify what could lead to this behavior, except if items were in deleted folders.

**Comment by qubez** [comment]
I first noticed that this happened on an install I put at a client - I just selected a few items, and many others were gone. I came to my test installation had previously deleted items also, made the screenshot, and the same effect - many more items disappeared than expected.

Perhaps the items should be shown here as a tree also, or a warning that the items xxxx must also be checked before the folder can be removed.

This would be bad if sysadmin thought they could first delete a folder they didn't recognize, but then put back other items that they knew weren't supposed to be deleted.
